radio/src/targets/simu/simpgmspace.cpp

```diff
@@ -1006,6 +1006,12 @@ FRESULT f_getcwd (TCHAR *path, UINT sz_path)
 
   // remove simuSdDirectory from the cwd
   strcpy(path, cwd + strlen(simuSdDirectory));
+
+  // convert backslashes to forward - issue #3024
+  for (int i = 0; path[i] != 0; i++) {
+	  if (path[i] == '\\') { path[i] = '/'; }
```

Comment:
Have a look in other files, you will see, here the indentation is KO (we use 2 chars) and a new line is missing